cms/blocks.py

```diff
@@ -11,3 +12,36 @@ class ResourceBlock(blocks.StructBlock):
 
     heading = blocks.CharBlock(max_length=100)
     detail = blocks.RichTextBlock()
+
+
+class InstructorBlock(blocks.StructBlock):
+    """
+    Block class that defines a instructor
+    """
+
+    name = blocks.CharBlock(
+        max_length=100, help_text="Name of the instructor.")
+    image = ImageChooserBlock(
+        help_text="Profile image size must be at least 300x300 pixels."
+    )
+    description = blocks.RichTextBlock(
+        help_text="A brief description about the instructor."
+    )
+
+
+class InstructorSectionBlock(blocks.StructBlock):
+    """
+    Block class that defines a instrcutors section
+    """
+    heading = blocks.CharBlock(
+        max_length=255,
+        help_text="The heading to display for this section on the product page.",
+    )
+    subhead = blocks.CharBlock(
+        max_length=255,
+        help_text="The subhead to display for this section on the product page.",
```

Comment:
I'd rather you remove mention of the product page because we are trying to create general purpose sections/child pages not specific to a page type. Don't want this to start getting confusing once we add this carousel or section to the home page.